### ISSUE TITLE
Doubts  loss_train Vs loss_val

### ISSUE THREAD

**Issue by carlotita22** [MIC-DKFZ/nnUNet#1537]
Hi everyone,

I have a doubt with respect to my results of loss,  I used tha data MSD task 2 (heart)

For 3D I used folder 0

![Imagen1](https://github.com/MIC-DKFZ/nnUNet/assets/83087352/b9a04032-b085-407e-a42b-f259a40d90bf)

For 2D I used folder 0

![Imagen2](https://github.com/MIC-DKFZ/nnUNet/assets/83087352/ad942128-3091-454a-b01c-6d30b9475e91)

Do you know how I could fix the difference between the training and validation curves?

Regards!

**Comment by saikat-roy** [comment]
@carlotita22 I am not sure what you mean by the difference between the curves, as they look okay to me. The `loss_tr` and `loss_val` are indicated on the y-axis on the left. The validation loss is higher than the training loss which is to be expected for a neural network. The `pseudo-dice` as indicated on the y-axis on the right indicates that a decent predictive performance is achieved for your task.
